apispec/apispec-model/src/main/scala/sttp/tapir/apispec/Schema.scala

```diff
@@ -18,6 +18,7 @@ case class Schema(
     writeOnly: Option[Boolean] = None,
     example: Option[ExampleValue] = None,
     deprecated: Option[Boolean] = None,
+    hidden: Option[Boolean] = None,
```

Comment:
Is there a `hidden` attribute in the OpenAPI schema? If not, we should remove this from here :)